test/specs/chrome.js

```diff
@@ -0,0 +1,49 @@
+import { describe } from '../utils/mochaw'
+import { asyncForEach } from '../utils/async'
+import { supportedLanguages } from '../utils/config'
+import { welcomeScenarios } from './scenarios/welcome'
+import { documentSelectorScenarios } from './scenarios/documentSelector'
+import { documentScenarios } from './scenarios/document'
+import { faceScenarios } from './scenarios/face'
+import { crossDeviceScenarios } from './scenarios/crossDevice'
+import { proofOfAddressScenarios } from './scenarios/proofOfAddress'
+import { modalScenarios } from './scenarios/modal'
+import { navigationScenarios } from './scenarios/navigation'
+
+const options = {
```

Comment:
I'm not sure if the specs to be used makes sense to be defined at such a high level.
I think it makes more sense to be set at a lower level, otherwise this is akin to saying init all screens (which is not necessarily a bad idea either).

Reply:
I agree!